pkg/policy/topology.go

```diff
@@ -65,7 +65,7 @@ func GetPlacementsFromTopologyPolicies(ctx context.Context, cli client.Client, a
 	}
 	hasTopologyPolicy := false
 	for _, policy := range policies {
-		if policy.Type == v1alpha1.TopologyPolicyType {
+		if policy.Type == v1alpha1.TopologyPolicyType && policy.Properties != nil {
```

Comment:
I would recommend directly raise errors here to notify user that he/she misuses the topology policy.

Reply:
that's better. accepted

Reply:
FYi, override policy also fit this change. :)

Reply:
override policy also have this change now. 

since env-binding is deprecated, it is not changed.